vm-setup/inventory.ini

```diff
@@ -1,2 +1,11 @@
 [virthost]
 localhost
+
+[virthost:vars]
+ansible_python_interpreter=/usr/bin/python3
+
+[bmh]
+192.168.111.20 ansible_user=ubuntu
```

Comment:
I don't think this should be statically added to the inventory, as this IP is configurable, and the user is distro specific?

Check https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/common/tasks/main.yml#L6

We could perhaps use add_host to add the bmh node by taking the first host from vm_nodes or something?

https://docs.ansible.com/ansible/latest/modules/add_host_module.html

Reply:
Thanks @hardys  , this is one of the reason why this PR is still as WIP.
add_host should work, I'm testing it.